sql_schema/Uber.sql

```diff
@@ -0,0 +1,57 @@
+CREATE DATABASE Cab;
+
+USE Cab;
+
+CREATE TABLE Customer(
+CustomerID INT PRIMARY KEY,
+CustomerName VARCHAR(70) NOT NULL,
+CustomerPhone CHAR(10) NOT NULL,
+CustomerEmail VARCHAR(60),
+CustomerAge INT ,
+CustomerGender CHAR(1),
```

Comment:
add normalization

Reply:
ALTER TABLE customer
DROP COLUMN Address;

CREATE TABLE Address(
AddressID INT PRIMARY KEY,
StateName VARCHAR(25) NOT NULL,
DistrictName VARCHAR(30) NOT NULL,
CityName VARCHAR(50) NOT NULL,
CustomerID INT,
FOREIGN KEY (CustomerID) REFERENCES Customer(CustomerID));